blog/en/blog/2023/02/09/secure-apisix-admin.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Securing Admin access to Apache APISIX
+authors:
+  - name: Nicolas Fränkel
+    title: Author
+    url: https://github.com/nfrankel
+    image_url: https://avatars.githubusercontent.com/u/752258
+keywords:
+  - Security
+  - Admin API
+description: >
+  API Gateways are critical components in one's infrastructure. If an attacker could change the configuration of routes, they could direct traffic to their infrastructure. Consequences could range from data theft to financial losses. Worse, data theft could only be noticed after a long time by mirroring the load. Hence, protecting your API Gateway is of utmost importance
```

Comment:
```suggestion
description: 
It's essential to secure API gateways, the critical components in infrastructures, to avoid data theft and financial losses.
```

Reply:
I don't understand how your suggestion improves the post. Could you please justify?

Reply:
🤔 Maybe because SEO wise 70-150 characters are ideal for descriptions. But I'm not sure if it actually makes a difference!

Reply:
That's the point: most SEO is based on assumptions only because Google keeps his algorithm secret to avoid people gaming it.

Reply:
> I don't understand how your suggestion improves the post. Could you please justify?

I shortened the description to make it concise and also for SEO because no more than 120 characters would be better. That's what we have done since Q4 last year. It works, although there are many other factors for our website growth.

Reply:
> 

Thanks for your kind suggestions. I'll read [Search Engine Optimization (SEO) Starter Guide](https://developers.google.com/search/docs/fundamentals/seo-starter-guide) carefully and discuss it with our team.